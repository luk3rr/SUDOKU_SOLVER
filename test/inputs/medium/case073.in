000050791 040000300 000000600 309086007 000700460 007000000 600830000 092000008 000100500 
