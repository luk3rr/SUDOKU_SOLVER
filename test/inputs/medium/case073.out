Solving the following grid:
0 0 0 | 0 5 0 | 7 9 1 
0 4 0 | 0 0 0 | 3 0 0 
0 0 0 | 0 0 0 | 6 0 0 
------+-------+------
3 0 9 | 0 8 6 | 0 0 7 
0 0 0 | 7 0 0 | 4 6 0 
0 0 7 | 0 0 0 | 0 0 0 
------+-------+------
6 0 0 | 8 3 0 | 0 0 0 
0 9 2 | 0 0 0 | 0 0 8 
0 0 0 | 1 0 0 | 5 0 0 


Solution found :')

2 8 6 | 3 5 4 | 7 9 1 
1 4 5 | 9 6 7 | 3 8 2 
9 7 3 | 2 1 8 | 6 4 5 
------+-------+------
3 1 9 | 4 8 6 | 2 5 7 
5 2 8 | 7 9 1 | 4 6 3 
4 6 7 | 5 2 3 | 8 1 9 
------+-------+------
6 5 1 | 8 3 2 | 9 7 4 
7 9 2 | 6 4 5 | 1 3 8 
8 3 4 | 1 7 9 | 5 2 6 

Algorithm: BFS
Total time: 476 ms
Total expanded states: 72867
